pkgs/top-level/darwin-aliases.nix

```
@@ -51,6 +51,10 @@ mapAliases ({
 
   builder = throw "'darwin.builder' has been changed and renamed to 'darwin.linux-builder'. The default ssh port is now 31022. Please update your configuration or override the port back to 22. See https://nixos.org/manual/nixpkgs/unstable/#sec-darwin-builder"; # added 2023-07-06
 
+  ### O ###
+
+  opencflite = throw "'darwin.opencflite' has been renamed to 'opencflite'"; # added 2024-05-02
```

Instead of throwing, would it be better to use alias it to `pkgs.opencflite`?

```suggestion
  opencflite = pkgs.opencflite; # added 2024-05-02
```